test/RESTAssured/pom.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.backbase</groupId>
+  <artifactId>local-backend-setup-restassured</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>RESTAssured</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
any particular reason to set older java versions?

Reply:
(Same as above) I think I must have messed up when setting the project up.
Sorry and thanks for spotting this